src/main/groovy/com/auth0/gradle/oss/LibraryPlugin.groovy

```diff
@@ -210,4 +212,45 @@ class LibraryPlugin implements Plugin<Project> {
             }
         }
     }
+
+    private void japiCmp(Project project) {
+        project.afterEvaluate {
+            def lib = project.extensions.oss
+            def baselineVersion = lib.baselineVersion
+            if (baselineVersion) {
```

Comment:
let's invert this condition and return sooner

Reply:
new commit to invert and return earlier pushed.